support-frontend/test/selenium/util/Browser.scala

```diff
@@ -20,6 +21,12 @@ trait Browser extends WebBrowser {
   def pageHasElement(q: Query): Boolean =
     waitUntil(ExpectedConditions.visibilityOfElementLocated(q.by))
 
+  // If there is more than 1 element matching the query then only one of them needs to be visible
+  def pageHasAtLeastOneVisibleElement(q: Query): Boolean = {
+    val elements = webDriver.findElements(q.by)
+    elements.asScala.exists(element => element.isDisplayed)
```

Comment:
how about adding some logging to confirm your theory?

Reply:
👍 it now logs:
`INFO selenium.util.Browser Visibility of element .stripe_checkout_app: List(true, false, false)`